graylog2-web-interface/src/theme/styles/formControlValidationStyles.jsx

```diff
@@ -0,0 +1,79 @@
+import { darken, lighten } from 'polished';
+
+import InputGroup from 'components/graylog/InputGroup';
+import teinte from '../teinte';
+import colorLevel from '../util/colorLevel';
+
+const VARIANTS = [
+  {
+    success: {
+      text: colorLevel(teinte.tertiary.tre, 6),
+      border: teinte.tertiary.tre,
+      background: colorLevel(teinte.tertiary.tre, -6),
+    },
+  },
+  {
+    warning: {
+      text: colorLevel(teinte.tertiary.sei, 6),
+      border: teinte.tertiary.sei,
+      background: colorLevel(teinte.tertiary.sei, -6),
+    },
+  },
+  {
+    error: {
+      text: colorLevel(teinte.secondary.uno, 6),
+      border: teinte.secondary.uno,
+      background: colorLevel(teinte.secondary.uno, -6),
+    },
+  },
+];
+
+const generateStyles = () => {
+  let styles = '';
+
+  VARIANTS.forEach((variant) => {
+    const key = Object.keys(variant)[0];
+
+    styles += `
```

Comment:
Let's add `styles += css` here.

Reply:
This is all removed in https://github.com/Graylog2/graylog2-server/pull/6982